vnext/ReactUWP/Views/RawTextViewManager.cpp

```diff
@@ -53,12 +53,15 @@ void RawTextViewManager::UpdateProperties(
     if (propertyName == "text") {
       run.Text(asHstring(propertyValue));
       if (nodeToUpdate->GetParent() != -1) {
-        auto parent = this->m_wkReactInstance.lock()
-                          ->NativeUIManager()
-                          ->getHost()
-                          ->FindShadowNodeForTag(nodeToUpdate->GetParent());
-        if (parent->m_children.size() == 1) {
-          auto view = static_cast<ShadowNodeBase &>(*parent).GetView();
+        auto instance = this->m_wkReactInstance.lock();
```

Comment:
It would be more readable like this:

if (auto instance = this->m_wkReactInstance.lock())
{
  if (auto parent = static_cast<ShadowNodeBase *>(
          instance->NativeUIManager()->getHost()->FindShadowNodeForTag(
              nodeToUpdate->GetParent())))
  {
    if (parent->m_children.size() == 1)
    {
      auto view = parent->GetView();
      auto textBlock = view.try_as<winrt::TextBlock>();
      if (textBlock != nullptr)
      {
        textBlock.Text(run.Text());
      }
    }}}

Reply:
I'm not used to the pattern of doing assignments inside conditions, usually that is a warning we have turned on as error in office code.  maybe the auto variable gets around that so the error can still be on to avoid accidental assignments